common/git.go

```diff
@@ -131,8 +138,10 @@ func findGitDirectory(fromFile string) (string, error) {
 		return gitPath, nil
 	} else if dir == "/" {
```

Comment:
On my windows machine this was showing up as `c:\` (lowercase).  Should we check for both?  Could these checks be merged into a single check ( line #139 + #141)

Reply:
It shows upper case on my machine.  In my case it stayed in the infinite loop with "c:\".  In any case the revised statement should cover it.

line: 139
} else if dir == "/" || dir == "C:\\" || dir == "c:\\" {